gitpod-io/website
# Remove individual plan from pricing page

## Description

Update pricing page following org-only mode and the removal of personal accounts.

| Before | After |
|--------|-------|
| ![Old - Plans and Pricing - Gitpod](https://user-images.githubusercontent.com/666176/236185635-1af0fc0b-4e85-4063-afa3-cd0387e787dc.png) | ![Plans and Pricing - Gitpod](https://user-images.githubusercontent.com/666176/236202866-3aa34404-6b27-4730-8519-1a6250344b2d.png) |

Note that now Enterprise is highlighted at the bottom table in order to align with the highlight at the top table.

## Related Issue(s)



Fixes WEB-295.

## How to test

Check [preview](https://deploy-preview-3617--gitpod-kumquat.netlify.app/pricing).

## Release Notes



```release-note
NONE
```
